xbmc/platform/win32/WIN32Util.h

```diff
@@ -110,4 +111,20 @@ class CWIN32Util
    * \return Formatted string
    */
   static std::string FormatHRESULT(HRESULT hr);
+
+private:
```

Comment:
private: can be inside TARGET_WINDOWS_DESKTOP bloc (not have empty private: for UWP)